### PR TITLE
Add repair step to fix messages where `in_reply_to` has `<>` as content

### LINKED ISSUE #5829: Create a repair step that removes botched thread root ids from the cache.
Ref: https://github.com/nextcloud/mail/pull/5828

### PR DESCRIPTION
**Pull request by miaulalala**
Fixes #5829

### CODE REVIEW

File: lib/Command/RepairMailTheads.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	/** @var LoggerInterface */
+	private $logger;
+
+	/** @var MailAccountMapper */
+	private $mapper;
+
+	/** @var MessageMapper */
+	private $messageMapper;
+
+	public function __construct(MailAccountMapper $mapper,
```
Comment (ChristophWurst) [inline lib/Command/RepairMailTheads.php:44]: nit: argument order does not match assignment order nor class property order :see_no_evil: 

I like the order of the argument list. adjust assignment below and props above accordingly

File: lib/Command/RepairMailTheads.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+		$this->mapper = $mapper;
+		$this->logger = $logger;
+		$this->messageMapper = $messageMapper;
+	}
+
+	/**
+	 * @return void
+	 */
+	protected function configure() {
```
Comment (ChristophWurst) [inline lib/Command/RepairMailTheads.php:57]:
```suggestion
	protected function configure(): void {
```

and drop the phpdoc

File: tests/Integration/Db/MessageMapperTest.php [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+			$performanceLogger
+		);
+
+		$qb = $this->db->getQueryBuilder();
+
+		$delete = $qb->delete($this->mapper->getTableName());
+		$delete->execute();
+	}
+
+	public function testResetInReplyTo() {
```
Comment (ChristophWurst) [inline tests/Integration/Db/MessageMapperTest.php:67]:
```suggestion
	public function testResetInReplyTo(): void {
```

File: tests/Integration/Db/MessageMapperTest.php [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+		$account = $this->createMock(Account::class);
+		$account->method('getId')->willReturn(13);
+		foreach (range(1, 10) as $i) {
```
Comment (ChristophWurst) [inline tests/Integration/Db/MessageMapperTest.php:70]: lend itself to use `array_map` :wink:

**Review by ChristophWurst — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Comment by miaulalala** [comment]
Had to remove the executeStatement from the MessageMapper.

**Pull request merged**